server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -707,17 +840,23 @@ public JobDetail healEntire(
     @Path("{owner_key}/activation_keys")
     @ApiOperation(notes = "Retrieves a list of Activation Keys for an Owner", value = "Owner Activation Keys")
     @ApiResponses({ @ApiResponse(code = 404, message = "Owner not found") })
-    public List<ActivationKey> ownerActivationKeys(
+    public List<ActivationKeyDTO> ownerActivationKeys(
```

Comment:
We should use this opportunity to upgrade this to the CandlepinQuery framework. We'll also need to update ActivationKeyCurator.listByOwner, but we'll be able to clean this method up substantially while getting some added benefits like paging and faster iteration.

Reply:
TODO on monday.